deploy-board/deploy_board/settings.py

```diff
@@ -86,6 +86,8 @@
 ENABLING_SERVICE_RATELIMIT_URL = os.getenv("ENABLING_SERVICE_RATELIMIT_URL", "OFF")
 KAFKA_MSGS_DELIVERED_METRIC = os.getenv("KAFKA_MSGS_DELIVERED_METRIC", "OFF")
 DASHBOARD_URL_ENDPOINT_FORMAT = os.getenv("DASHBOARD_URL_ENDPOINT_FORMAT","OFF")
+CONSOLE_PROJECT_URL_FORMAT = os.getenv("CONSOLE_PROJECT_URL_FORMAT", "OFF")
+CONSOLE_PROJECT_DATA_URL_FORMAT = os.getenv("CONSOLE_PROJECT_DATA_URL_FORMAT", "OFF")
```

Comment:
Can we refer to these as `NIMBUS_PROJECT...` since the project info actually comes from Nimbus. Also, we already have some Nimbus-related code in the repo (e.g. the Nimbus client)

Reply:
Isn't this exposing internal Pinterest code? Especially the nimbus client. This seems like a huge security risk.